test/e2e/specs/default-config/recordedit/add-defaults.spec.js

```diff
@@ -168,7 +177,37 @@ describe('Record Add with defaults', function() {
             expect(foreignKeyInput.getText()).toBe(values.foreign_key_value, "Foreign key input default is incorrect");
             expect(foreignKeyDisabledInput.getText()).toBe(values.foreign_key_disabled_value, "Foreign key disabled default is incorrect");
         });
-
+        //mdhelp page
```

Comment:
`default-config` is more specifically meant to test features that are affected by config options when those options are not defined. In this case, the markdown page will always show up regardless of any configuration. It almost doesn't matter what config set this test goes in. So we should focus on putting this test in a spec that seems appropriate.

With that being said, we actually don't have a test that is meant to just test the presentation of the `recordedit` app. We test that presentation as part of the add spec (`/specs/all-features-confirmation/recordedit/add.spec.js`). This seems like the most appropriate place for this test case.